### DISCUSSION TITLE
Is v2.9.1 ready to be used?

### DISCUSSION THREAD

**Discussion by alexghr** [kriszyp/lmdb-js#264] [Q&A] [answered]
Hi, just started using the library and noticed that the latest release/tag on Github is v2.9.1 but on npm it's still 2.8.5 with 2.9.1 released under the `dev` tag. Is 2.9.1 still being tested or should it be promoted to `latest` on npm?

**Comment by kriszyp** [comment, marked as answer]
I moved the latest tag to 2.9.1, thank you for reminding me.
